src/sandstorm/run-bundle.c++

```diff
@@ -652,6 +652,17 @@ public:
                   .build();
             },
             "For internal use only.")
+        .addSubCommand("admin-token",
+            [this]() {
+              return kj::MainBuilder(context, VERSION,
+                      "Generates a new admin token that you can use to access the admin settings "
+                      "page. This is meant for initial setup, or if an admin account is locked out.")
+                  .addOption({"short"}, [this]() { shortOutput = true; return true; },
```

Comment:
Typically this would be called "quiet", with 'q' as a short option: {'q', "quiet"}